docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+<cycle-end type = "global garbage collect"/> (cycle ends)
+
+<allocation-satisfed/>                      (required allocation has been achieved)
+
+<exclusive-end>                             (STW pause ends)
+```
+
+The following example shows a `balanced` global cycle that is triggered during a [global mark cycle](vgclog_examples.md#balanced-global-mark-gc-cycle). 
+
+The start of the GMP work processing subincrement of the global mark cycle, which runs concurrently with application threads, is recorded using the `<concurrent-start>` element.
+
+```xml
+<concurrent-start id="2009" type="GMP work packet processing" contextid="2003" timestamp="2021-03-05T12:16:43.109">
+  <concurrent-mark-start scanTarget="18446744073709551615" />
+</concurrent-start>
+```
+
+After the start of the concurrent subincrement, The logs record an allocation failure using `<af-start>`. The `<concurrent-end>` element attribute `terminationReason` shows that a termination of the concurrent increment was requested by the garbage collector. 
+
+```xml
+<af-start id="2010" threadId="00000000008AA780" totalBytesRequested="24" timestamp="2021-03-05T12:16:43.109" intervalms="1212.727" />
+<concurrent-end id="2011" type="GMP work packet processing" contextid="2003" timestamp="2021-03-05T12:16:43.110" terminationReason="Termination requested">
+<gc-op id="2012" type="mark increment" timems="0.893" contextid="2003" timestamp="2021-03-05T12:16:43.110">
+  <trace-info scanbytes="584612" />
+</gc-op>
+</concurrent-end>
+```
+
+The next element, the `<cycle-continue>` element, records information about the switch of cycle type from a global mark cycle, recorded as type `global mark phase`, to a global cycle, recorded as type `global garbage collect`.
+
+<cycle-continue id="2013" oldtype="global mark phase" newtype="global garbage collect" contextid="2003" timestamp="2021-03-05T12:16:43.110" />
+
+A global cycle increment is recorded by `<gc-start>` and has the same `contextid` as the global mark cycle's elements. The global cycle operations are run during a STW pause and as a modification to the global mark cycle rather than a completely new cycle. The memory snapshot within the `<gc-start>` element is taken before the global increment's operations run and can be compared with a similar snapshot that is taken afterward to understand the effect on the heap.
+
+```xml
+<gc-start id="2014" type="global garbage collect" contextid="2003" timestamp="2021-03-05T12:16:43.110">
+  <mem-info id="2015" free="0" total="838860800" percent="0">
+    <mem type="eden" free="0" total="524288" percent="0" />
+    <remembered-set count="12832" freebytes="33331072" totalbytes="33382400" percent="99" regionsoverflowed="0" regionsstable="0" regionsrebuilding="1593"/>
+  </mem-info>
+</gc-start>
+```
+
+At the start of the global cycle's increment, the amount of memory available in the heap is zero. In some cases, the amount of memory will be close to full, and in other cases, the memory will be full.
```

Comment:
'the amount of memory will be close to full' does not sound right to me as for example 'heap will be close to full'.

or you can stick with the complement (amount of free memory) in the whole paragraph (as with the first sentence).